package.json

```diff
@@ -2,7 +2,7 @@
   "name": "lightning-app",
   "version": "0.1.0",
   "description": "Lightning Wallet Application",
-  "homepage": "https://lightning.engineering",
+  "homepage": "./",
```

Comment:
The npm docs say this attribute is for the project homepage? https://docs.npmjs.com/files/package.json#homepage

Reply:
[https://github.com/facebook/create-react-app/blob/ca7d227ae0354e04941ba0e4f76c0bcbf3050dfa/config/webpack.config.prod.js#L20-L23](url) create-react-app uses `package.json`'s `homepage` field when it creates the webpack config as webpack's `publicPath`, which is what the static files are served from. Got this fix from here: https://github.com/facebook/create-react-app/issues/165 lmk if it isn't acceptable